aminator/plugins/cloud/ec2.py

```diff
@@ -358,16 +397,37 @@ def register_image(self, *args, **kwargs):
         if vm_type == 'hvm':
             if context.ami.get("enhanced_networking", False):
                 ami_metadata['sriov_net_support'] = 'simple'
+            if context.ami.get("ena_networking", False):
+                ami_metadata['ena_networking'] = True
```

Comment:
Since `ami_metadata` can take a boolean, unless there's a chance for the `ena_networking` key to have a non-boolean value, you can simplify this to
`ami_metadata['ena_networking'] = context.ami.get('ena_networking', False)`